samples/templates/express-embedded-auth-with-sdk/web-server/views/authenticator-webauthn-verify.mustache

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    {{>head}}
+    <script src="/okta-auth-js.min.js" type="text/javascript"></script>
+    <script src="/assets/js/webauthn.js" type="text/javascript"></script>
+  </head>
+  <body id="samples">
+  
+    {{>menu}}
+
+    <div class="okta-sign-in-form ui center aligned relaxed grid">
+      <div class="row">
+        <div id="page-title-header" class="ui header">
+          <h2>{{title}}</h2>
+        </div>
+      </div>
+
+      <div class="row">
+        {{>formMessages}}
+      </div>
+
+      {{#challengeData}}
+        <div class="row">
+          <form id="challenge-authenticator-form" class="ui large form" action="{{action}}" method="POST">
+
+            <input id="credentials-id" type="hidden" name="id" autoComplete="off" required />
+            <input id="credentials-clientData" type="hidden" name="clientData" autoComplete="off" required />
+            <input id="credentials-authenticatorData" type="hidden" name="authenticatorData" autoComplete="off" required />
+            <input id="credentials-signatureData" type="hidden" name="signatureData" autoComplete="off" required />
+            
+            <div class="row">
+              <div id="credentials-error" class="ui negative message"></div>
+            </div>
+
+            <div class="row">
+              <button id="retry-button" class="ui fluid large button" onclick="askCredentials()" type="button">
+                Retry
+              </button>
+
+              <button id="submit-button" class="ui fluid large primary submit button" type="submit">
+                Verify
+              </button>
+            </div>
+          </form>
+        </div>
+
+        <script type="text/javascript">
+          const challengeData = {{{challengeData}}};
+          const authenticatorEnrollments = {{{authenticatorEnrollments}}};
+
+          const askCredentials = async () => {
+            try {
+              const options = buildCredentialRequestOptions(challengeData, authenticatorEnrollments);
```

Comment:
same as above, if possible let's try to keep the webauthn related logic transparent to devs.

Reply:
maybe put it under crypto/webauthn